### PR TITLE
Update staging.studio.code.org links to studio.code.org ones (warning…

### PR DESCRIPTION
**Pull request by bcjordan**
…: includes levelbuilder file changes, will be manually re-seeded)

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/sitemap.haml [hunk 1]
```diff
@@ -13,15 +13,15 @@ title: Sitemap
...
-      %a{href:'http://studio.code.org/s/1/level/1'} K-8 Intro    
+      %a{href:'http://studio.code.org/s/1/level/1'} K-8 Intro
       &bull;
       %a{href:'http://studio.code.org/flappy/1'} Flappy
       &bull;
-      %a{href:'http://staging.studio.code.org/s/course1'} course1
+      %a{href:'http://staging-studio.code.org/s/course1'} course1
       &bull;
-      %a{href:'http://staging.studio.code.org/s/course2'} course2
+      %a{href:'http://staging-studio.code.org/s/course2'} course2
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/sitemap.haml]: All of these should be https://, right?

File: dashboard/config/scripts/levels/Multi Strings Images 1.level [hunk 2]
```diff
@@ -27,7 +27,7 @@
       }
     ],
     "title": "Strings and Images Multiple Choice",
-    "content1": "http://staging.studio.code.org/script_assets/msm/green-rect.png",
+    "content1": "http://studio.code.org/script_assets/msm/green-rect.png",
```
Comment (islemaster) [inline dashboard/config/scripts/levels/Multi Strings Images 1.level]: Are we sure that these assets are available in the production environment?  I don't know how that pipeline works.

Comment (islemaster) [inline dashboard/config/scripts/levels/Multi Strings Images 1.level]: I can at least hit this one at the new URL, so it's good.

File: pegasus/sites.v3/code.org/public/sitemap.haml (hunk 1 above)
Comment (bcjordan) [inline pegasus/sites.v3/code.org/public/sitemap.haml]: Shoot that's right, I'll re-do

File: dashboard/config/scripts/levels/Multi Strings Images 1.level (hunk 2 above)
Comment (bcjordan) [inline dashboard/config/scripts/levels/Multi Strings Images 1.level]: Yep, if the changes are no longer sitting on staging they should be on prod

**Comment by islemaster** [comment]
I think you need to change http:// to https:// on all the staging-studio URLs, otherwise looks fine.

**Pull request merged**